content/reference/rest-api/templates/create.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Templates: create'
+weight: 10
+
+menu:
+  main:
+    name: 'create'
+    identifier: 'rest-api-templates-create'
+    parent: 'rest-api-templates'
+    pre: 'Creates a new template.'
+---
+
+Creates a new template.
+
+# Request
+
+## HTTP Request
+
+```
+POST https://cawemo.com/api/v1/templates
+```
+
+## Request Body
+
+In the request body, supply a [Templates resource]({{< ref "/reference/rest-api/templates/_index.md#resource-representation" >}}) with the following properties:
+
+<table class="table table-striped">
+  <tr>
+    <th>Property Name</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>catalogId</td>
+    <td>string</td>
+    <td>Unique identifier of the catalog project the template should be created in.</td>
+  </tr>
+  <tr>
+    <td>content</td>
+    <td>object</td>
+    <td>JSON definition of the template. See <a href="https://github.com/camunda/camunda-modeler/tree/master/docs/element-templates#defining-templates">the Camunda Modeler documentation</a> for a detailed description.</td>
+  </tr>
+  <tr>
+    <td>content.name</td>
+    <td>string</td>
+    <td>Name of the template.</td>
+  </tr>
+</table>
+
+# Response
```

Comment:
I already prepared a separate page that includes a list of error status codes and messages but refrained from adding it to this PR as it would just be a stub at this time, I agree with you that this is out of scope for the first iteration.

I don't know if it makes too much sense to explicitly mention the different variants of 2xx status codes for every resource method - I'd rather prefer mentioning at a central location (somewhere below "Overview") that any 2xx response should be handled as success and only discuss 4xx and 5xx errors in more detail.

What do you think?

Reply:
Sounds good, and I agree, 4xx and 5xx are more important to clarify.